### PR TITLE
Changelog: add latest

### PR DESCRIPTION
**Pull request by Guts**
Add a shortcut to automatically return the latest version item from the changelog using the `latest` keyword.

Considering this changelog:

```markdown
# CHANGELOG

### 0.1.2 - 23/07/2020

* Tag without "v" prefix
* Add a CHANGELOG.md file for testing

### v0.1.1 - 22/07/2020

* Tag with a "v" prefix to check the regular expression
* Previous version

### 0.1.0 - 21/07/2020

* Very old version

###
```

The command: `qgis-plugin-ci changelog latest` would return:

```bash
* Tag without "v" prefix
* Add a CHANGELOG.md file for testing
```

### CODE REVIEW

**Review by Gustry — APPROVED** [review]
Thanks, LGTM ;-)

Can you add the test in https://github.com/opengisch/qgis-plugin-ci/blob/master/test/test_changelog.py ?

**Comment by Guts** [comment]
> Thanks, LGTM ;-)

/me has just learned a new acronym.

> Can you add the test in https://github.com/opengisch/qgis-plugin-ci/blob/master/test/test_changelog.py

I will.

**Comment by Guts** [comment]
@Gustry @3nids  could you tell me how to launch tests? I tried with nose2 but tests are failing:

```log
----------------------------------------------------------------------
Ran 10 tests in 0.064s

FAILED (failures=5, errors=3)
```

I see in the Travis configuration that a Transifex and a GitHub token are set up before. Is it required?  
Is there a way to run the tests locally?

**Comment by Gustry** [comment]
I think you can just care about the changelog test.
Indeed some tests will fail on your local repo because you will need github or transifex token as well.

**Comment by Guts** [comment]
Yes, I saw it and edited my previous comment). So it's done and ready.

File: scripts/qgis_plugin_ci.py [hunk 1]
```diff
@@ -40,7 +40,11 @@ def main():
 
     # changelog
     changelog_parser = subparsers.add_parser('changelog', help='gets the changelog content')
-    changelog_parser.add_argument('release_version', help='The version to be released')
+    changelog_parser.add_argument('release_version',
+                                  help='The version to be released. If nothing is speficied, \
+                                      the latest version specified into the changelog is used.',
+                                  default="latest"
+                                  )
```
Comment (Gustry) [inline scripts/qgis_plugin_ci.py:47]: Sorry, the readme.md is updated as well with this command output. Can you update it ?

**Review by Gustry — APPROVED** [review]

**Comment by Gustry** [comment]
@Guts After this PR, let us know if you want a new version or if some other PR are coming soon.

**Comment by Guts** [comment]
> @Guts After this PR, let us know if you want a new version or if some other PR are coming soon.

I won't have time to work on the regex but I would like to propose a PR about a real documentation (GitHub Pages or RTD), not only the README which is quite hard to read.

File: scripts/qgis_plugin_ci.py (hunk 1 above)
Comment (Guts) [reply scripts/qgis_plugin_ci.py:47]: done.

**Comment by Gustry** [comment]
I thought that Travis will run as soon this work is merged in the main repo, but it seems that Travis is dead : https://travis-ci.com/github/opengisch/qgis-plugin-ci :(

**Comment by 3nids** [comment]
On opengisch we don't have any time left on Travis.
I need to move everything to Github workflows. I'll try to do this later this week!

**Comment by 3nids** [comment]
OK, tests are running on new CI :)
Let me we know when we try to release!

**Comment by Gustry** [comment]
Thanks @3nids for the quick detravisication !

**Comment by Guts** [comment]
> OK, tests are running on new CI :)
> Let me we know when we try to release!

Woo so quick! Thanks!

**Pull request merged**